installer/scripts/omsadmin.sh

```diff
@@ -604,7 +605,7 @@ onboard()
         cleanup_certs
         return $INVALID_PROXY
     elif [ $error -ne 0 ]; then
-        log_error "Error during the onboarding request: curl returned $error. Check the correctness of the workspace ID and shared key or run omsadmin.sh with '-v'"
+        log_error "Error during the onboarding request: curl returned $error when trying to connect to $CURL_URL . Check the correctness of the workspace ID and shared key, as well as the connectivity to the url, or run omsadmin.sh with '-v' to get verbose output"
```

Comment:
- is there a command we can specify to run here to check connectivity to the URL? for instance `telnet ${WORKSPACE_ID}.oms.${URL_TLD} 443`? admittedly I am not positive what the most robust command to run would be, but you should be able to test out by restricting the NSG and/or installing a firewall (have seen this commonly ... would be good to account for that not just NSG) and then seeing what commands are a reliable indicator of connectivity.
- nit no space after `$CURL_URL`